wp-content/plugins/ig-menu-help/lang/integreat-help-de_DE.po

```diff
@@ -1,2 +1,8 @@
 msgid "Integreat Help"
 msgstr "Integreat Hilfe"
+
+msgid "Mark as read"
+msgstr "Als gelesen markieren"
+
+msgid "Mark as not read"
+msgstr "Als nicht gelesen markieren"
```

Comment:
To me personally, "Mark as unread"/"Als ungelesen markieren" sounds a bit more natural than "Mark as not read"/"Als nicht gelesen markieren", but that's just a minor detail.